### DISCUSSION TITLE
Will Yarp bindings and iCub bindings for python change the previous functions and complied project?

### DISCUSSION THREAD

**Discussion by YANGGANG1971** [robotology/community#389] [Q&A] [answered]
Recently, I want to try to use python to control the physical iCub names iCubOsaka01.
But I found that in the current version in our lab's PCs, there are no Yarp bindings and iCub bindings. 
So I want to install it. 

While installing the Yarp bindings and iCub bindings, will these affect the existing demos like Emotion_Demo, Green_Ball, etc. ? 

Just referring to [the tutorial link](http://wiki.icub.org/wiki/Python_bindings#iCub_bindingsl), is it enough? Do I have other stuff to notice?

**Comment by pattacini** [comment, marked as answer]
Hi @YANGGANG1971 

Bindings go alongside the standard build products, thus if you compile bindings you won't overwrite your current builds.
